### PR TITLE
[Backport release-8.2.0] fix(backup/gcs): listing backups when no basePath is set

### PR DESCRIPTION
**Pull request by lenaschoenburg**
Manual backport of https://github.com/camunda/zeebe/pull/12221 to fix merge conflicts in the parent pom.

### CODE REVIEW

File: backup-stores/gcs/pom.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
   <name>Zeebe Backup Store for GCS</name>
 
+<<<<<<< HEAD
```
Comment (Zelldon) [inline backup-stores/gcs/pom.xml:24]: @oleschoenburg I think this is not expected :D

**Review by ChrisKujawa — CHANGES_REQUESTED** [review]

File: backup-stores/gcs/pom.xml (hunk 1 above)
Comment (oleschoenburg) [reply backup-stores/gcs/pom.xml:24]: Yikes :+1:

Comment (oleschoenburg) [reply backup-stores/gcs/pom.xml:24]: That's what I get from trying to merge without a merge tool.

**Review by ChrisKujawa — APPROVED** [review]
LGTM

**Pull request merged**